### ISSUE TITLE
TOPPView does not open mzid files

### ISSUE THREAD

**Issue by hroest** [OpenMS/OpenMS#2232]
I have tried to open mzid files directly in TOPPView and it fails with the message:

```
Cannot add a dataset that contains no survey scans. Aborting!
```

so it seems that TOPPView does not understand the mzid data format. What is the currently recommended way of loading identifications into TOPPView, does one have to convert them to idxml first?

**Comment by hroest** [comment]
The same is true for the "annotate with identification" feature: 

```
Unknown file type. No annotation performed.
```

**Comment by hroest** [comment]
the fix turned out to be quite easy: #2234 - is there a reason we did not do this before (e.g. political) or is mzIdentML not stable enough so that it may lead to bad user experiences? 

The background here is that a user was asking whether OpenMS can handle mzIdentML and I said "sure" without checking and then it turned out that TOPPView cant...

**Comment by jpfeuffer** [comment]
Pull request was merged. Can this be closed?
